module-1/web-project/your-code/WebScraping-DogAdoption.ipynb

```diff
@@ -18,7 +18,14 @@
    "cell_type": "markdown",
```

Comment:
You could have displayed dogs photos from the url. It could be something like this: 
```Python
#You need to install and import any of the libraries of Python for displaying images. 
from IPython.display import Image, HTML
def path_to_image_html(path):
return '<img src="'+ path + '" style=max-height:124px;"/>'
HTML(df.to_html(escape=False ,formatters=dict(column_name_with_image_links=path_to_image_html)))
```
Using PIL library is another option
Take a look on this Kaggle post. 
https://www.kaggle.com/stassl/displaying-inline-images-in-pandas-dataframe
If you have time, try it in your code.